include/igl/slice_into.cpp

```diff
@@ -34,7 +34,7 @@ IGL_INLINE void igl::slice_into(
 #endif
 
   // create temporary dynamic sparse matrix
-  Eigen::DynamicSparseMatrix<T, Eigen::RowMajor>  dyn_Y(Y);
```

Comment:
This is an incorrect change. Was it tested? I guess it will be significantly slower. `DynamicSparseMatrix` is an obsolete class that had much better performance. Changing this to `SparseMatrix` doesn't make sense with the current code and introduces _at least_ a copy and possible a lot of overhead.

Reply:
Coming back to #801, I feel we should really try to get rid of this `DynamicSparseMatrix` at some point. I haven't had time to come up with a benchmark test case however, but I can try to give it a shot next week.

Reply:
ok i revert that change. I changed it because it was deprecated...